### PR TITLE
Fix incremental task samples tests

### PR DESCRIPTION
**Pull request by wolfs**
the problem was that the tasks changing
the inputs of the incremental task did
not declare the inputs as outputs, so
Gradle did not pick up the changes.

Fixes gradle/gradle-private#3264.

### CODE REVIEW

File: subprojects/docs/src/snippets/tasks/incrementalTask/groovy/build.gradle [hunk 1]
```diff
@@ -9,6 +10,7 @@ task originalInputs() {
 
 // tag::updated-inputs[]
 task updateInputs() {
+    outputs.dir('inputs')
```
Comment (blindpirate) [inline subprojects/docs/src/snippets/tasks/incrementalTask/groovy/build.gradle:13]: Do you mean `inputs.dir('inputs')`?

**Review by blindpirate — COMMENTED** [review]
Are they typos?

File: subprojects/docs/src/snippets/tasks/incrementalTask/groovy/build.gradle (hunk 1 above)
Comment (wolfs) [reply subprojects/docs/src/snippets/tasks/incrementalTask/groovy/build.gradle:13]: No. The outputs of this task is the directory `inputs`. It changes files in there.

**Review by blindpirate — APPROVED** [review]
LGTM!

**Pull request merged**